index.js

```diff
@@ -28,6 +28,13 @@ app.post('/generate', (req, res) => {
   })
 })
 
+app.get('/qrcards', (req, res) => {
+  const cardQuery = knex.select().table('qrcodes')
+
+  cardQuery.then((data) => res.json(data))
+    .catch((err) => console.log(err))
```

Comment:
if there's an error,  you should probably return a `500` status code, along with an error message. The general convention I like to follow is to send an object `{ error: [error message here] }`. To tack on a status code on something you want to return, you can easily chain them:

```js
res.status(500).json({ error: 'Error retrieving qr codes.' })
```